_posts/2019-02-10-cadena-responsabilidades.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "cadena-responsabilidades"
+date: 2019-02-10
+author: Esteban Galicia
+tags: python patrones SOLID Responsability Chain
+comments: true
+excerpt: "Los patrones de diseño como eje central en el desarrollo de software"
```

Comment:
Este campo debe ser un resumen de lo que el post trata o algo que haga que los usuarios quieran leer el post para tu post le pondría algo com "Como el patrón de diseño Chain of Responsibility te puede ayudar a crear mejor código" o "Aprende a usar el patrón ..."